presets/4.5/filters/defaults.txt

```diff
@@ -0,0 +1,48 @@
+#$ TITLE: Default 4.5 Filter settings
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: defaults, filter, filters, reset
+#$ AUTHOR: Betaflight
+#$ DESCRIPTION: Resets Filter settings to 4.5 defaults
+
+set gyro_hardware_lpf = NORMAL
+set gyro_lpf1_type = PT1
+set gyro_lpf1_static_hz = 250
+set gyro_lpf2_type = PT1
+set gyro_lpf2_static_hz = 500
+set gyro_notch1_hz = 0
+set gyro_notch1_cutoff = 0
+set gyro_notch2_hz = 0
+set gyro_notch2_cutoff = 0
+
+defaults group_id 554 nosave # PG_DYN_NOTCH_CONFIG
+
+set gyro_lpf1_dyn_min_hz = 250
+set gyro_lpf1_dyn_max_hz = 500
+set gyro_lpf1_dyn_expo = 5
+set gyro_filter_debug_axis = ROLL
+
+set acc_lpf_hz = 25
+
+set dshot_bidir = OFF
```

Comment:
This is no longer default for all targets as more will be enabled by default on flash.

Reply:
Yes just asking - not asking to enable - but maybe not use it.

Reliance for other presets should not be an issue. If it's on already it does not fail when asking again.

Reply:
> Yes just asking - not asking to enable - but maybe not use it.
> 
> Reliance for other presets should not be an issue. If it's on already it does not fail when asking again.

we can remove it from here after fixing other presets. Need to go through them one-by one and see which ones are relying on this one be OFF by default and add `set dshot_bidir = OFF` there.
But let's make it the last step, so we don't make a mess. First need to copy all presets straight up without modifications. And then individually go one by one.